src/Library/demos/Color Dialog/main.blp

```diff
@@ -0,0 +1,51 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Color Dialog";
+  description: _("Show a dialog to select a color");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+    spacing:42;
+
+    Box{
+      orientation:vertical;
+
+      ColorDialogButton color_dialog_button {
```

Comment:
I think having the default color here `white` makes it hard to see what it is at a glance. Let's add a solid default color like blue, red, etc.